R/guides-.R

```diff
@@ -279,6 +279,10 @@ Guides <- ggproto(
 
     # Merge and process layers
     guides$merge()
+    if (isTRUE(get_key)) {
+      return(lapply(guides$params, `[[`, "key"))
+    }
+
```

Comment:
I thought it was cleaner to early-exit out of `guides$build()` than to re-implement the build code.